hal/tests/TESTS/mbed_hal_fpga_ci_test_shield/CMakeLists.txt

```diff
@@ -1,13 +0,0 @@
-# Copyright (c) 2020 ARM Limited. All rights reserved.
-# SPDX-License-Identifier: Apache-2.0
```

Comment:
Could we remove this file in its own separate commit please, ensuring the commit message explains the reasoning.